build.sbt

```diff
@@ -214,7 +214,9 @@ lazy val `kyo-os-lib` =
         .dependsOn(`kyo-core` % "test->test;compile->compile")
         .settings(
             `kyo-settings`,
-            libraryDependencies += "com.lihaoyi" %% "os-lib" % "0.10.1"
+            scalacOptions --= Seq(
+                "-language:strictEquality"
```

Comment:
It'd be nice if we could keep this flag. If you have a type that should allow equality checks, it's possible to add `derives CanEqual` to it. In case you want to disable strict equality for a scope, it's possible to use `given [T]: CanEqual[T, T] = CanEqual.derived`.